### ISSUE TITLE
New Enhancement: Stock Exchange trade

### ISSUE THREAD

**Issue by ravirajlondhepatil** [freqtrade/freqtrade#2204]
Hello Sir,

can we modify Freqtrade to apply same logic to Stock exchange and automate it?

thanks in advance

with best regrads,
Ravi L

**Comment by hroff-1902** [comment]
no, till the ccxt library used in freqtrade to communicate with exchanges will support stock exchanges.
and they are not going to move this direction
